ui-frontend/packages/catalog-ui-search/package.json

```diff
@@ -30,7 +30,7 @@
     "url": "https://github.com/codice/ddf.git"
   },
   "engines": {
-    "node": ">=0.10.5"
+    "node": ">=16.16.0"
```

Comment:
❓ I'm using node version 17.3.1.  For the node commands in `package.json` I've had to add the `--openssl-legacy-provider` argument.  Is it save to use `>=16.16.0` here without that argument?